app/services/green_lanes/fetch_subheadings_service.rb

```diff
@@ -0,0 +1,21 @@
+module GreenLanes
+  class FetchSubheadingsService
+    ITEM_ID_LENGTH = 10
+    def initialize(goods_nomenclature_item_id)
```

Comment:
We had this discussion about writing a service class with a constructor that needs all the parameters for the service method. In this way, you need to create an instance of service class for each request and one instance can only process one goods_nomenclature_item_id. Is there a real reason for following this pattern in ruby?

Reply:
Generally in OOP, you pass constructor parameters, when the service cannot do it's job without those instances or values passed in.

Reply:
Good point. In this case, I think that's fine since we are going to expand the service, and we'll need `goods_nomenclature_item_id` from some private methods.